### PR TITLE
Adds a license

### PR DESCRIPTION
**Pull request by orta**
General homework for OSS

### CODE REVIEW

**Comment by orta** [comment]
Don't want it to end up in https://github.com/artsy/potential/issues/157

**Comment by ashkan18** [comment]
haha! nice and :fast:

**Comment by orta** [comment]
it's safe to say that I've done this before

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2018 Askan Nasseri, Artsy Inc
```
Comment (alloy) [inline LICENSE]: @orta  @ashkan18 Who is this ‘Askan’ person? ;)

Comment (ashkan18) [reply LICENSE]: hehe nice catch 👓that would be me :) will fix it in my upcoming PR

**Pull request merged**